workflows/prognostic_c48_run/docs/local-execution.rst

```diff
@@ -77,6 +77,27 @@ the end of the previous one will be appended. For example, the following for loo
 
         runfv3 append gs://path/to/failing/run
 
+.. note::
+
+    An option "mpi_launcher" has been added to the subcommand "append". This option 
+    allows users to configure the segmented runs on High Perfromance Computing
+    systems and cloud platfroms. The supported values are "mpirun" and "srun".
+   
+    An example of runnning model with 5 segments on the cloud platform:
+
+    for i in {1..5}
+    do
+       runfv3 append --mpi_launcher mpirun gs://bucket/prognostic_run
+    done
```

Comment:
I think we do not necessarily need to cover this case, since it is covered above (using the default value for the option).
```suggestion
```